cypress/e2e/API/api.cy.js

```diff
@@ -0,0 +1,8 @@
+/// <reference types="cypress" />
+describe('API Tests', () => {
+    it('Realiza uma request para a pagina e verifica se a resposta é um statusCode 200 ', () => {
+        cy.request('/').should(function ({ status }) {
```

Comment:
Ao executar este _request_, você está "batendo" na URL do _frontend_, não do _backend_.

Reply:
Além disso, que tal usar _arrow function_ em vez de `function`, para manter o padrão, visto que os blocos `describe` e `it` usam _arrow function_ (`() => {}`)

Reply:
O mesmo vale para outros lugares onde está utilizando `function`.